src/index.tsx

```diff
@@ -180,7 +181,7 @@ const buildApp = async () => {
 }
 
 buildApp().then(() => {
-    // do absolutely nothing
+    enablePatches();
```

Comment:
Should be moved outside the callback to ensure patches are working in the buildApp function